src/database/sequlize-handler/migrations/04_webhooks.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require('sequelize');
+const uuid = require('uuid');
+const { WEBHOOK_EVENT_TYPES } = require('../../../common/consts');
+
+const tableName = 'webhooks';
+const webhookEventsTableName = 'webhook_events';
+const webhookEventMappingTableName = 'webhook_events';
+const columns = [
+    {
+        name: 'name',
+        dt: Sequelize.DataTypes.TEXT('medium')
+    },
+    {
+        name: 'global',
+        dt: Sequelize.DataTypes.BOOLEAN
+    }
+];
+
+async function takeActionOnColumn(describedTable, newColumnName, existAsyncAction, notExistAsyncAction) {
+    if (describedTable[newColumnName]) {
+        return existAsyncAction();
+    }
+    return notExistAsyncAction();
+}
+
+function createEnumRow(name) {
+    return {
+        name,
+        id: uuid(),
+        created_at: new Date(),
+        updated_at: new Date()
+    };
+}
+
+module.exports.up = async (query, DataTypes) => {
+    let describedWebhooks = await query.describeTable(tableName);
+    const webhooksEventTypes = WEBHOOK_EVENT_TYPES.map(createEnumRow);
+    const promises = [
+        ...columns.map(({ name, dt }) =>
+            takeActionOnColumn(
+                describedWebhooks,
+                name,
+                () => null,
+                () => query.addColumn(tableName, name, dt)
+            )
+        ),
+        takeActionOnColumn(describedWebhooks, 'url', () => query.renameColumn(tableName, 'url', 'webhook_url'), () => null)
```

Comment:
let's remove this, i think we should leave the field name as `url` since it's in the `webhooks` table

Reply:
done